locales/en.json

```
@@ -14,7 +14,8 @@
         "sourceCode": "Source Code",
         "faq": "FAQ",
         "error": "Error",
-        "contributors": "Contributors"
+        "contributors": "Contributors",
+        "contributorsDescription": "vrc-get and ALCOM are a open-source project made with contributions from many developers. Here are some contributors to the project:"
```

Gramatical error *(a instead of an)*
```suggestion
        "contributorsDescription": "vrc-get and ALCOM are an open-source project made with contributions from many developers. Here are some contributors to the project:"
```